src/main/docker/gitlab-jenkins-mysql.yml

```diff
@@ -9,27 +9,33 @@ services:
             - ./gitlab/gitlab-local-setup.sh:/gitlab-local-setup.sh
         environment:
             - GITLAB_OMNIBUS_CONFIG=prometheus_monitoring['enable'] = false; gitlab_rails['gitlab_shell_ssh_port'] = 2222; gitlab_rails['monitoring_whitelist'] = ['0.0.0.0/0']
+            - GITLAB_ROOT_PASSWORD
         ports:
             - "2222:22"
             - "8081:80"
         hostname: gitlab
         networks:
-            artemis:
+            artemis-gitlab:
                 ipv4_address: 172.33.0.2
     jenkins:
         build: jenkins
+        user: root
         volumes:
             - artemis-jenkins-data:/var/jenkins_home
+            - ./jenkins/jenkins-disable-csrf.groovy:/var/jenkins_home/init.groovy # Disable CSRF token
+            - ./jenkins/master-key-only-for-testing.key:/var/jenkins_home/master.key # Preset master key to use pre-generated secrets
+            - ./jenkins/jenkins-casc-config.yml:/usr/share/jenkins/ref/jenkins-casc-config.yml:ro
             - /var/run/docker.sock:/var/run/docker.sock
-            - /usr/local/bin/docker:/usr/bin/docker
-            - ./jenkins/jenkins-casc-config.yml:/usr/share/jenkins/ref/jenkins-casc-config.yml
+            - /usr/bin/docker:/usr/bin/docker:ro
+            - /usr/bin/com.docker.cli:/usr/bin/com.docker.cli
```

Comment:
Where does `com.docker.cli` come from? I’ve not seen that one before.

Reply:
You are right, I'm also not sure where it is coming from, I removed it.